tests/engagement/test_churn.py

```diff
@@ -1,207 +1,23 @@
 import functools
-import arrow
 
 import pytest
 from pyspark.sql import functions as F
-from pyspark.sql.types import (
-    StructField, StructType, StringType,
-    LongType, IntegerType, BooleanType
-)
 
-from mozetl.churn import churn, schema
+from data import SUBSESSION_START, WEEK_START_DS, generate_dates, \
```

Comment:
Importing from `data` is an implicit relative import that are discouraged (and won't work on Python 3.x), use instead `from .data import ....`.